lib/tooling/run.cpp

```diff
@@ -106,6 +108,25 @@ int run(evmc::VM& vm,
 
     const auto result = vm.execute(host, rev, msg, exec_code.data(), exec_code.size());
 
+    // gas-cost-estimator decoration
+    //
+    for (unsigned int run_id = 0; run_id < sample_size; ++run_id) {
+        std::chrono::steady_clock::time_point start_time, end_time, end_time_timer;
+        start_time = std::chrono::steady_clock::now();
+
+        const auto result = vm.execute(host, rev, msg, exec_code.data(), exec_code.size());
```

Comment:
This will include time for code analysis (EVMC API does not provide a way to separate these yet).

Reply:
that's fine, I think Łukasz and myself opted to do it like this.